plugins/modules/junos_logging.py

```diff
@@ -198,7 +262,6 @@ def main():
         rotate_frequency=dict(type="int"),
         size=dict(type="int"),
         files=dict(type="int"),
-        src_addr=dict(),
```

Comment:
Looks like an existing `key` is removed?

Reply:
yes, It was dead key.